test/optimization/test_stable_set.py

```diff
@@ -52,14 +52,14 @@ def test_stable_set(self):
     def test_stable_set_vqe(self):
         """ VQE Stable set  test """
         result = VQE(self.qubit_op,
-                     RYRZ(self.qubit_op.num_qubits, depth=3, entanglement='linear'),
+                     EfficientSU2(reps=3, entanglement='linear'),
                      L_BFGS_B(maxfun=6000)).run(
                          QuantumInstance(BasicAer.get_backend('statevector_simulator'),
                                          seed_simulator=aqua_globals.random_seed,
                                          seed_transpiler=aqua_globals.random_seed))
-        x = sample_most_likely(result['eigvecs'][0])
-        self.assertAlmostEqual(result['energy'], -29.5)
-        self.assertAlmostEqual(result['energy'] + self.offset, -25.0)
+        x = sample_most_likely(result.eigenstate)
+        self.assertAlmostEqual(result.eigenvalue, -29.5, places=5)
```

Comment:
I imagine the places needed to be reduced here due most likely to #954

Reply:
Yes I think that's the same cause